setup.cfg

```diff
@@ -25,6 +25,7 @@ filterwarnings =
     ignore:Writing of electrodes.tsv is not supported for kind.*:RuntimeWarning
     ignore:numpy.ufunc size changed.*:RuntimeWarning
     ignore:tostring\(\) is deprecated.*:DeprecationWarning
+    ignore:Bids root is None and full BIDS filepath cannot be automatically inferred.*:RuntimeWarning
```

Comment:
I would rather catch the warnings. These can be legit issues

Reply:
Okay let me take a look at when this should go off... I'll try some things later tonight.